docs/source/project_documentation/advanced_user_topics/language_extensions.md

```diff
@@ -8,63 +8,42 @@ For AaC users who find that existing definitions are insufficient or that they w
 
-[Example gen_protobuf.yaml L45-L51](https://github.com/DevOps-MBSE/AaC/blob/3dfc11baf0ec8d7250608bd16caebdae837fc8af/python/src/aac/plugins/first_party/gen_protobuf/gen_protobuf.yaml#L45-L51)
-```yaml
-ext:
-   name: SchemaMessageOptions
-   type: schema
-   schemaExt:
-      add:
-        - name: protobuf_message_options
-          type: KeyValuePair[]
+```{eval-rst}
+.. literalinclude:: ../../../../python/src/aac/plugins/first_party/gen_protobuf/gen_protobuf.yaml
+    :language: yaml
+    :lines: 85-91
 ```
 
 You can see that this extension `SchemaMessageOptions` is extending `schema` with an additional field `protobuf_message_options`. So, the AaC DSL now supports defining schema definitions with the additional field like so:
-```yaml
-schema:
-    name: MyMessage
-    protobuf_message_options:
-        - key: java_package
-          value: com.example.foo
-    fields:
-        - name: exampleContent
-          type: string
+
+```{eval-rst}
+.. literalinclude:: ../../../../python/model/protobuf_flow/Data.aac
+    :language: yaml
+    :lines: 5-13
+    :emphasize-lines: 5
 ```
 
 ## Extending an Enum
 The AaC DSL also supports extending enum values. These extensions can be used to add additional enumeration values. The first-party plugin `Gen-Protobuf` leverages an enum extension to add new primitive types that are supproted in Protobuf5, but aren't present in the [core spec](https://github.com/DevOps-MBSE/AaC/blob/main/python/src/aac/spec/spec.yaml).
 
-
-[Example gen_protobuf.yaml new primitive types extension](https://github.com/DevOps-MBSE/AaC/blob/main/python/src/aac/plugins/first_party/gen_protobuf/gen_protobuf.yaml#L26-L43)
-```yaml
-ext:
-  name: ProtobufPrimitiveTypesExtension
-  type: Primitives
-  enumExt:
-    add:
-      - double
-      - float
-      - int32
-      - int64
-      - uint32
-      - uint64
-      - sint32
-      - sint64
-      - fixed32
-      - fixed64
-      - bool
-      - string
-      - bytes
+```{eval-rst}
+.. literalinclude:: ../../../../python/src/aac/plugins/first_party/gen_protobuf/gen_protobuf.yaml
+    :language: yaml
+    :lines: 30-47
 ```
 
 With these new primitive types, users of the `Gen-Protobuf` plugin can model protobuf-specific data structure fields. For instance, users can model inter-service messages like so:
 
-```yaml
-schema:
-    name: MyMessage
-    fields:
-        - name: exampleContent
-          type: string
-        - name: dataValue
-          type: fixed64
+```{eval-rst}
+.. literalinclude:: ../../../../python/model/protobuf_flow/Data.aac
+    :language: yaml
+    :lines: 41-47
+    :emphasize-lines: 6-7
+```
```

Comment:
Okay, so you wanted to include two examples instead of one - got it. I'm okay with this it just wasn't obvious that this is what you were doing since there was only one example, previously.

Reply:
Okay, so you wanted to include two examples instead of one - got it. I'm okay with this it just wasn't obvious that this is what you were doing since there was only one example, previously.